src/lesson2/HomeWorkApp.java

```diff
@@ -0,0 +1,96 @@
+package lesson2;
+
+import java.util.Scanner;
+
+public class HomeWorkApp {
+
+    private static boolean sum(int a, int b){
```

Comment:
пробелы более явно разделяют синтаксические части
```suggestion
    private static boolean sum(int a, int b) {
```
```suggestion
    private static boolean sum(int a, int b){
```